### PR TITLE
Travis auto build dist

### PR DESCRIPTION
**Pull request by crouchygt**
Automatically compile the dist css via travis ci

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
 node_js:
-  - "0.12"
+- '0.12'
 before_script:
-  - npm install
+- npm install
 script:
-  - npm test
+- npm test
+- npm dist
```
Comment (lukejpreston) [inline .travis.yml]: npm run dist

Comment (crouchygt) [inline .travis.yml]: You don't need to specify run

Comment (lukejpreston) [inline .travis.yml]: it just failed...

**Comment by lukejpreston** [comment]
Once you have this working and committed feel free to close #24

**Comment by crouchygt** [comment]
@lukejpreston I am going to remove the dist folder from the project and update the instructions to grab everything from the release

**Pull request merged**